content/en/synthetics/platform/private_locations/configuration.md

```diff
@@ -27,6 +27,11 @@ docker run --rm datadog/synthetics-private-location-worker --help
 ```
 synthetics-pl-worker.exe --help
 ```
+{{% /tab %}}
+{{% tab "Kubernetes" %}}
+
+Check example on [github.com/DataDog/helm-charts/tree/main/charts/synthetics-private-location][6]
```

Comment:
```suggestion
Refer to the example in the [Datadog Helm repository][1].

[1]: https://github.com/DataDog/helm-charts/tree/main/charts/synthetics-private-location
```

Reply:
Since the link is in a tab, the URL should be included within the tab as well.

Reply:
Correct me if I'm wrong, but readme says user should add that repo to Helm first, so maybe a link to the readme would be more beneficial than just to chart.yaml file.